utilities/tags/addTagsUsingCSV.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import requests
+
+SEMGREP_API_KEY = ""
+API_URL = "https://semgrep.dev/api/v1/deployments/{deployment-slug}/projects/{project_name}/tags"
```

Comment:
Probably want to provide a way to get the deployment slug automatically too, either from the CLI or from the endpoint using the token. Same again for all the files.